core-library/src/main/java/org/silverpeas/core/admin/service/Admin.java

```diff
@@ -2735,44 +2732,57 @@ public String identify(String sKey, String sSessionId, boolean isAppInMaintenanc
       List<String> result = new ArrayList<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "result" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1481)